casper/src/main/scala/coop/rchain/casper/dag/BlockDagKeyValueStorage.scala

```diff
@@ -327,4 +363,62 @@ object BlockDagKeyValueStorage {
       stores.deployPool,
       stores.invalidBlocks
     )
+
+  def deployStatus[F[_]: Sync](
```

Comment:
`deployStatus` is only used in `BlockDagKeyValueStorage`. Why not make it a private member of this class? Then there is no need to pass `execMap` and `expiredMap`. In fact, many function parameters are fields of the `BlockDagKeyValueStorage` class.

Reply:
https://github.com/rchain/rchain/pull/3682#discussion_r869618823, but ok I'll move it